CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v4.3.1 (not published yet)
```

Comment:
Shall we roll this up into the upcoming 4.4.0 or are we planning to publish a patch before then?
![image](https://user-images.githubusercontent.com/29152776/118534921-cf00d080-b717-11eb-8da0-9578c8e7ca1c.png)

Reply:
I think it might be best to adjust the roadmap so 4.4.0 is mobile-friendly use menu, 4.5.0 is collapsible appbar.  The latter will probably take a while.  I have no issues making 4.4.0 include this bug fix, the drawer tooltip hover prop, and the user-menu mobility enhancements.